packages/onchainkit/src/fund/components/FundButton.test.tsx

```diff
@@ -36,6 +36,12 @@ vi.mock('@/internal/hooks/useTheme', () => ({
   useTheme: vi.fn(),
 }));
 
+vi.mock('../../wallet/components/ConnectWallet', () => ({
```

Comment:
seems like a weird change in this PR

Reply:
aha, i see, makes sense thanks. in the future it'd be nice if you left a comment to that effect so that this type of change didn't seem like an unintentional artifact 🙏